client/data/application-passwords/use-app-passwords-query.js

```diff
@@ -0,0 +1,12 @@
+import { useQuery } from 'react-query';
+import wp from 'calypso/lib/wp';
+
+const useAppPasswordsQuery = () =>
+	useQuery( [ 'application-passwords' ], () => wp.req.get( '/me/two-step/application-passwords' ), {
+		refetchOnWindowFocus: false,
+		select( data ) {
```

Comment:
Should we have this be `enabled` only if the current user is logged in?

Reply:
I don't know. Are you even allowed to access `/me/security/two-step` in case you're not logged in? My testing shows you're being redirected to `/log-in` if not logged in.

Reply:
That's true. I think I was being overly cautious here.